docs/guides/guide.md

```diff
@@ -76,7 +82,7 @@ Go to [http://localhost:8080/](localhost:8080) to see our application showing a
 
 ## Adding Bootstrap
 
-DoneJS makes it easy to import other projects that are published on [NPM](https://npmjs.org). In this section we will install and add [Bootstrap](http://getbootstrap.com/) to the page and see DoneJS's [hot module swapping](./Features.html#hot-module-swapping) in action.
+DoneJS makes it easy to import other projects that are published on [NPM](https://npmjs.org). In this section, we will install and add [Bootstrap](http://getbootstrap.com/) to the page, and see DoneJS's [hot module swapping](./Features.html#hot-module-swapping) in action.
```

Comment:
I don't think the second comma is needed since the list only has two items
- install and add bootstrap to the page
- see hot module swapping in action